app/controllers/katello/api/v2/alternate_content_sources_controller.rb

```diff
@@ -92,6 +108,10 @@ def find_smart_proxies
       if params[:smart_proxy_ids]
         @smart_proxies = ::SmartProxy.where(id: params[:smart_proxy_ids])
         fail HttpErrors::NotFound, _("Couldn't find smart proxies with id '%s'") % params[:smart_proxy_ids].to_sentence if @smart_proxies.empty?
```

Comment:
probably don't need this line anymore. The code below should be able to handle 0 records.